reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -163,24 +169,30 @@ private void addRepresentations(Statistics summary, Map<String, String> rowData,
          rowData.put(operationName + ".Requests", String.valueOf(defaultOutcome.requests));
          rowData.put(operationName + ".Errors", String.valueOf(defaultOutcome.errors));
          rowData.put(operationName + ".ResponseTimeMax", String.valueOf(defaultOutcome.responseTimeMax));
-         rowData.put(operationName + ".ResponseTimeMean", String.valueOf(defaultOutcome.responseTimeMean));
+         rowData.put(operationName + ".ResponseTimeMean", String.valueOf(round(defaultOutcome.responseTimeMean, 2)));
       }
       OperationThroughput throughput = os.getRepresentation(OperationThroughput.class, TimeUnit.MILLISECONDS.toNanos(summary.getEnd() - summary.getBegin()));
       if (throughput != null) {
-         rowData.put(operationName + ".ThroughputGross", String.valueOf(throughput.gross));
-         rowData.put(operationName + ".ThroughputNet", String.valueOf(throughput.net));
+         rowData.put(operationName + ".ThroughputGross", String.valueOf(round(throughput.gross, 1)));
+         rowData.put(operationName + ".ThroughputNet", String.valueOf(round(throughput.net, 1)));
+      }
+      for (double percentile : percentiles) {
+         Percentile result = os.getRepresentation(Percentile.class, percentile);
+         if (result != null) {
+            rowData.put(operationName + ".RTM_" + percentile, String.valueOf(round(result.responseTimeMax, 2)));
+         }
       }
       MeanAndDev meanAndDev = os.getRepresentation(MeanAndDev.class);
       if (meanAndDev != null) {
-         rowData.put(operationName + ".ResponseTimeMean", String.valueOf(meanAndDev.mean));
-         rowData.put(operationName + ".ResponseTimeDeviation", String.valueOf(meanAndDev.dev));
+         rowData.put(operationName + ".ResponseTimeMean", String.valueOf(round(meanAndDev.mean, 2)));
```

Comment:
You should rather specify the total number of valid digits than arbitrary decimal spaces.